stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -121,7 +121,7 @@ public struct UnownedJob: Sendable {
   @available(SwiftStdlib 9999, *)
   @_alwaysEmitIntoClient
   @inlinable
-  public func runSynchronously(isolated serialExecutor: UnownedSerialExecutor,
```

Comment:
aligning this with some discussions on forums in other threads about "`isolated`" parameters.

FYI @rjmccall @hborla 

(this is new API coming in for the TaskExecutors)